### ISSUE TITLE
Are account_updates no longer part of websocket stream with trade_updates?

### ISSUE THREAD

**Issue by geauxvirtual** [alpacahq/user-docs#104]
It seems in this commit that looks to have been pushed directly to master, https://github.com/alpacahq/user-docs/commit/4898d607be071eedda21afc2121d2dbf81363d58, any information regarding account_updates as part of the websocket stream has been removed.

Are account_updates no longer part of the websocket stream? Is there a reason the above commit went in without a PR with explanation for removal of the account_updates section?

**Comment by d-e-s-o** [comment]
Yeah, I suggest you get used to it. It's just the usual for Alpaca. See https://github.com/alpacahq/Alpaca-API/issues/75 and the response that Alpaca managed to produce.
